backend/slackbot/worker.js

```diff
@@ -351,7 +378,13 @@ async function report(statistics){
 
     // Build the slack message to send back with attachments
     message += `Report for:\nApp: ${name}\nVersion: ${version}\n`;
-    message += `Between ${startDate} and ${endDate}, sentiment has been mostly ${attitude} for ${Object.keys(rawReviews).length} reviews`;
+    message += `Between ${startDate} and ${endDate}, sentiment has been mostly ${attitude} for ${Object.keys(rawReviews).length} `;
```

Comment:
There's a stat for this now - instead of using `rawReviews` you can request the `numReviews` stat and get this information directly.